engine/src/main/java/org/destinationsol/game/projectile/ProjectileConfigs.java

```diff
@@ -38,40 +41,46 @@
     public ProjectileConfigs(OggSoundManager soundManager, EffectTypes effectTypes, GameColors cols) {
         myConfigs = new HashMap<>();
 
-        Json json = Assets.getJson("core:projectilesConfig");
-        JsonValue rootNode = json.getJsonValue();
+        Set<ResourceUrn> projectileConfigurationFiles = Assets.getAssetHelper().list(Json.class,
+                "[a-z]*:projectilesConfig");
```

Comment:
There are way longer lines than this in the PR. Either break them all into parts, or don't break this one.